spec/spec_helpers/webmock_helpers.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module WebmockHelpers
+  def webmock_requests
```

Comment:
believe it or not, this is not functionality easily exposed to the user in WebMock.